README.md

```diff
@@ -181,6 +181,7 @@ Unless otherwise noted these projects are open-source with permissive licensing:
 - [LLMFarm](https://github.com/guinmoon/LLMFarm?tab=readme-ov-file) (MIT)
 - [KanTV](https://github.com/zhouwg/kantv?tab=readme-ov-file)(Apachev2.0 or later)
 - [Dot](https://github.com/alexpinel/Dot) (GPL)
+- [MindMac](https://mindmac.app) (proprietary, offer free version)
```

Comment:
Many proprietary apps in this list offer free versions, there is no need to mention that.

```suggestion
- [MindMac](https://mindmac.app) (proprietary)
```

Reply:
@slaren Thank you for your review. I've just pushed your suggestion. Please help to review again!